stage1/init/mount.go

```diff
@@ -0,0 +1,199 @@
+// Copyright 2014 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// functions for creating a mount units for
+// managing inner(kind=empty) and external(kind=host) volumes
+// important: used only for kvm flavor (lkvm based)
+// idea:
+// for example when we have two volumes:
+// --volume=foo,kind=empty
+// --volume=bar,kind=host,source=/host/bar
+// in stage1/rootfs rkt creates two folders (it is rootfs of guest)
+// 1. /mnt/foo - which is mounted through 9p host thanks to lkvm --9p=/host/bar,bar flag shared to quest
+// 2. /mnt/bar - is created as empty directory in guest
```

Comment:
Above, `foo` is for `empty` and `bar` is for `host`. There is a mismatch.

I suggest calling the volumes `hostdata` and `temporary` in the examples to avoid confusion.